Core/RadioHandler.cpp

```diff
@@ -137,15 +144,30 @@ bool RadioHandlerClass::Init(fx3class* Fx3, void (*callback)(void*context, const
 		DbgPrintf("WARNING no SDR connected\n");
 		break;
 	}
-	adcrate = adcnominalfreq;
 	hardware->Initialize(adcnominalfreq);
```

Comment:
shall we give adcrate a reasonable default in case the caller didn't call SetSampleRate

Reply:
Thanks. I'll revert this change.

Reply:
Hmmm...
It's already set to `adcnominalfreq` in constructor. That's why I have removed this line.